merlin/models/tf/inputs/embedding.py

```diff
@@ -380,6 +385,14 @@ def call(
         else:
             out = self._call_table(inputs, **kwargs)
 
+        if self.l2_batch_regularization_factor > 0:
+            if isinstance(out, dict):
```

Comment:
In this case where we have a dict with potentially more than one feature. Do we want to call `add_loss` more than once? Currently it's going to apply this only to the last feature.

If we want to call it for each feature, then either need to loop through the feature names again here or could put this code inside the `_call_table` method?